Configuration/AlCa/python/autoCond.py

```diff
@@ -64,19 +64,19 @@
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in PEAK mode
     'phase1_2018_cosmics_peak'     : '122X_upgrade2018cosmics_realistic_peak_v1',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2021
-    'phase1_2021_design'           : '122X_mcRun3_2021_design_v8',
+    'phase1_2021_design'           : '122X_mcRun3_2021_design_v9',
     # GlobalTag for MC production with realistic conditions for Phase1 2021
-    'phase1_2021_realistic'        : '122X_mcRun3_2021_realistic_v8',
+    'phase1_2021_realistic'        : '122X_mcRun3_2021_realistic_v9',
     # GlobalTag for MC production (cosmics) with realistic conditions for Phase1 2021,  Strip tracker in DECO mode
-    'phase1_2021_cosmics'          : '122X_mcRun3_2021cosmics_realistic_deco_v8',
+    'phase1_2021_cosmics'          : '122X_mcRun3_2021cosmics_realistic_deco_v9',
     # GlobalTag for MC production with realistic conditions for Phase1 2021 detector for Heavy Ion
-    'phase1_2021_realistic_hi'     : '122X_mcRun3_2021_realistic_HI_v8',
+    'phase1_2021_realistic_hi'     : '122X_mcRun3_2021_realistic_HI_v9',
     # GlobalTag for MC production with realistic conditions for Phase1 2023
-    'phase1_2023_realistic'        : '122X_mcRun3_2023_realistic_v8',
+    'phase1_2023_realistic'        : '122X_mcRun3_2023_realistic_v9',
     # GlobalTag for MC production with realistic conditions for Phase1 2024
-    'phase1_2024_realistic'        : '122X_mcRun3_2024_realistic_v8',
+    'phase1_2024_realistic'        : '122X_mcRun3_2024_realistic_v9',
     # GlobalTag for MC production with realistic conditions for Phase2
-    'phase2_realistic'             : '122X_mcRun4_realistic_v5'
+    'phase2_realistic'             : '122X_mcRun4_realistic_v6'
```

Comment:
```suggestion
    'phase2_realistic'             : '122X_mcRun4_realistic_v9'
```

seems this one does not exist see [log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ab5f37/22379/unitTests/failed.html)

Reply:
Yes, this was a typo, I did it correctly in the description